Sources/BowEffects/Data/Internal/Dictionary+Extensions.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+func -<K: Hashable, V>(lhs: Dictionary<K, V>, rhs: K) -> Dictionary<K, V> {
+    var copy = lhs
+    copy.removeValue(forKey: rhs)
+    return copy
+}
+
+func +<K: Hashable, V>(lhs: Dictionary<K, V>, rhs: (K, V)) -> Dictionary<K, V> {
+    var copy = lhs
+    copy[rhs.0] = rhs.1
+    return copy
+}
+
+extension Dictionary {
+    var arrayValues: [Value] {
+        return self.map { x in x.value }
```

Comment:
Apple gives you something similar... self.values -> `Array(values)`

Reply:
Replaced.